templates/default/tensorboard-cleanup.sh.erb

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+TENSORBOARD_DIR="$1"
+
+help() {
+    echo ""
+    echo "usage: $0 TENSORBOARD_DIR"
+    echo ""
+    exit 1
+}
+
+if [ $# -ne 1 ]; then
+  help
+fi
+
+if [ ! -d "$TENSORBOARD_DIR" ] ; then
+    echo "Secret dir does not exist: $TENSORBOARD_DIR"
+    exit 2
+fi
+
+# Remove the TensorBoard directory in the staging directory
+if [ $TENSORBOARD_DIR != "" ] ; then
```

Comment:
@robzor92 this could be one clause with an `AND` operator.

Reply:
Fixed